data/templates/openvpn/server.conf.tmpl

```diff
@@ -127,6 +127,16 @@ push "dhcp-option DNS6 {{ nameserver }}"
 {%     if server.domain_name is defined and server.domain_name is not none %}
 push "dhcp-option DOMAIN {{ server.domain_name }}"
 {%     endif %}
+{%     if server['2fa'] is defined and server['2fa'] is not none %}
+{%       if server['2fa']['totp'] is defined and server['2fa']['totp'] is not none %}
+plugin "/usr/lib/openvpn/openvpn-otp.so" "otp_secrets=/config/auth/openvpn/{{ ifname }}-otp-secrets otp_slop=
+{{- server['2fa']['totp']['slop']|default(180) }} totp_t0=
```

Comment:
Jinja2 `-` syntax is no longer required as all templates are rendered using the trim_blocks statement.

Reply:
Also it looks that you back-assign a variable to the line above - please use:

```
otp_step= {{ server.2fa.totp.step }}
totp_digits= {{ server.2fa.totp.digits }}
```

If you have values requireing default values, see the `<defaultValue>` XML definition. A good start would be:
* https://github.com/vyos/vyos-1x/blob/current/interface-definitions/ssh.xml.in#L123-L136
* https://github.com/vyos/vyos-1x/blob/current/src/conf_mode/ssh.py#L49-L55

OpenVPN already uses a ton of default values from the XML definition.